### ISSUE TITLE
private repo dependency

### ISSUE THREAD

**Issue by rpoleski** [jdeast/EXOZIPPy#6]
`gridsearches.py` starts with `import sfit_minimizer`, which is a private repo of @jenniferyee. This has to be changed so that everybody can install and run EXOZIPPY.

**Comment by jenniferyee** [comment]
Yup. This is on my list. Apparently, I already have the beginnings of a draft paper on this.

**Comment by rpoleski** [comment]
This is unclear to me. You're going to first write a paper and then allow others to run EXOZIPPy tests?

I'd like to suggest to remove relative imports (which are discouraged in PEP8).

It's hard for me to contribute, if I cannot run the code :(

**Comment by jenniferyee** [comment]
Pretty sure I shared the private repo with you already, but I can check on
Monday.
--------------------------------------------------------------
Dr. Jennifer C. Yee
(she/her)

Office: P-341
Center for Astrophysics | Harvard & Smithsonian
60 Garden St, MS-15
Cambridge, MA 02138
--------------------------------------------------------------

**Comment by rpoleski** [comment]
Yes, I have the access to that private repo. And I was able to run one of the unit test files.

The issue persists - an outside user is not able to run the code. Possible solution:
 - release `sfit_minimizer` right away,
 - make a copy of `sfit_minimizer` that has only capabilities needed by EXOZIPPy, or
 - hide somehow EXOZIPPy code that calls `sfit_minimizer` e.g., in a separate branch.

**Comment by jdeast** [comment]
sfit_minimizer is no longer necessary in EXOZIPPy (it was broken off into MMEXOFAST), but also, sfit_minimizer is now public. Closing.
